utils/config-utils.js

```diff
@@ -1,5 +1,26 @@
 import { version } from '_/package.json';
 
+async function initDevtools() {
+  try {
+    // Check devtools availability
+    const response = await fetch('/api/devtools-health');
```

Comment:
Why go to the backend here instead of the devtools directly? Like why not just

```
const response = await fetch('localhost:8098');
if (response.status !== 200) {
  throw new Error('Devtools not available');
}
```

Reply:
That doesn't work due to a CORS issue, so we need to proxy it. I also don't want the full script, only a true/false boolean as a health directive.